moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -67,6 +67,8 @@ const EditStaffView = () => {
       password,
       roles,
       workgroup_id,
+      is_user_group_member,
+      note,
```

Comment:
this fixes a bug i introduced [here](https://github.com/cityofaustin/atd-moped/pull/1096). it is not currently possible to edit these two user fields in Moped unless the user a non-login user :/